modules/apt.sh

```diff
@@ -61,26 +61,32 @@ _apt_write_list_file() {
     local repo_url="$2"
 
     cat >"$repo_file" <<EOF
-deb https://${repo_url}/ubuntu ${SERIES} main
-# deb-src https://${repo_url}/ubuntu ${SERIES} main
+deb ${repo_url}/ubuntu ${SERIES} main
+# deb-src ${repo_url}/ubuntu ${SERIES} main
 EOF
 }
 
 _apt_add_auth() {
-    local repo_host="$1"
+    local repo_path="$1"
     local credentials="$2"
 
     local login="${credentials%:*}"
     local password="${credentials#*:}"
     [ -f "$APT_AUTH_FILE" ] || touch "$APT_AUTH_FILE"
     chmod 600 "$APT_AUTH_FILE"
-    echo "machine ${repo_host}/ubuntu/ login ${login} password ${password}" \
+    echo "machine ${repo_path}/ubuntu/ login ${login} password ${password}" \
          >>"$APT_AUTH_FILE"
 }
 
 _apt_remove_auth() {
-    local repo_host="$1"
-
-    # shellcheck disable=SC1117
-    sed -i "/^machine ${repo_host}\/ubuntu\/ login/d" "$APT_AUTH_FILE"
+    local repo_path="$1"
```

Comment:
Like my other comment, should we make this function take a full url as an argument, and strip the protocol as needed? repo_path should just be a path, not a hostname plus a path, in my mind.